data/clinvar_cpg_concept_ids.tsv

```diff
@@ -0,0 +1,470 @@
+C2608083
```

Comment:
would probably prefer this to be `.txt` since it is just a list.

Reply:
this is now a `.txt` file